crates/starknet-devnet/tests/benches/mint_bench.rs

```diff
@@ -0,0 +1,36 @@
+use criterion::{black_box, criterion_group, criterion_main, Criterion, SamplingMode};
+use tokio::runtime::Runtime;
+
+use crate::common::background_devnet::BackgroundDevnet;
+
+#[path = "../common/mod.rs"]
+pub mod common;
+
+static DUMMY_ADDRESS: u128 = 1;
+static DUMMY_AMOUNT: u128 = 1;
+
+async fn mint_iter(f: &str) {
+    let devnet =
+        BackgroundDevnet::spawn_with_additional_args(&[
+            format!("--state-archive-capacity={}", f).as_str()
+        ])
+        .await
+        .expect("Could not start Devnet");
+
+    for _n in 1..5000 {
```

Comment:
Why run it `4999` times and not e.g. `5000` times?

Reply:
you can use 1..=5000

Reply:
already changed in 68e46e93f1a7738ee8c37f3d82c3623fca2ccefd